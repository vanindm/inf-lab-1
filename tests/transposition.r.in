2 3
1.5 2.1 3.51345
6543.1345 2.123416 1.51345
