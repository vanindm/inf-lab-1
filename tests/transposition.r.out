1.500000 6543.134500 
2.100000 2.123416 
3.513450 1.513450 
